### DISCUSSION TITLE
PrimeVue menus submenu is hidden behind the PrimeVue datatable

### DISCUSSION THREAD

**Discussion by ferguson03** [primefaces/community#3221] [PrimeVue]
I have a very simple screen layout which has a primevue/menubar at the top and a  primevue/Datatable below it.    The sub menu of the primevue/menubar is rendered  behind the DataTable almost as if the z-index is messed up. 

example here:
![image](https://github.com/user-attachments/assets/4b4f8aac-de01-42bc-837f-3190c5717565)

In the example above there are two more menu items which cannot be seen because they are being rendered behind the primevue DataTable

I have narrowed down this issue by inspecting the css removing styles
primevue applies a  class called p-datatable-scrollable-table  to the table when the scrollable property is enabled. When this occurs this problem happens.  If you remove the p-datatable-scrollable-table class off the DataTable then it renders fine as shown here.

![image](https://github.com/user-attachments/assets/e8a50d00-c1c4-4c98-b70b-4b8823bec27e)

I am running the latest as of today  12/15/24  - version 4.25

**Comment by ferguson03** [comment]
I did find out that it is the thead option of the datatable that causes this because the thead of the datatable is set to sticky.  When it is set to sticky the issue arises.   So to resolve I hacked it for now overriding the primevue css class of the submenu like this and giving it a top level z-index.   This fixes the issue


`ul.p-menubar-submenu{z-index:500}`

**Reply by renewestenberg** [reply]
Thanks, fixed my issue as well.
